src/data/vortex.graphql

```diff
@@ -1449,6 +1439,7 @@ type Query {
     Returns the last _n_ elements from the list.
     """
     last: Int
+    userId: String
```

Comment:
FYI: I just generated the new schema and copy-pasted it here. Probably this was a change that wasn't reflected yet.